packages/frontend/app/components/app/sidebar/index.hbs

```diff
@@ -1,13 +1,80 @@
 <aside ...attributes>
   <div>
+    {{did-insert this.switchToContacts}}
```

Comment:
I need to remove did-insert, actually. But this can be replaced by just setting the default values on the class